templates/taxi/car_detail.html

```diff
@@ -3,20 +3,31 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  
+    <form method="post" action="{% url 'taxi:toggle-driver' car.id %}">
+    <h1>Drivers
+      {% csrf_token %}
+      {% if user in car.drivers.all %}
+        <input type="submit" value="Delete me from this car" class="btn btn-danger link-to-page ">
+      {% else %}
+        <input type="submit" value="Assign me to this car" class="btn btn-success link-to-page ">
+      {% endif %}
+    </h1>
```

Comment:
The `<h1>` tag is used to wrap the form and its contents, which is semantically incorrect. The `<h1>` tag should be used for headings only. Consider using a `<div>` or another appropriate container element instead of `<h1>` to wrap the form content.